src/managers/ApplicationCommandManager.js

```diff
@@ -177,6 +204,139 @@ class ApplicationCommandManager extends BaseManager {
       default_permission: command.defaultPermission,
     };
   }
+
+  /**
+   * Fetches the permissions for one or multiple commands.
+   * <warn>When calling this on ApplicationCommandManager, guildID is required.
+   * To fetch all permissions for an uncached guild use `fetchPermissions(undefined, '123456789012345678')`</warn>
+   * @param {ApplicationCommandResolvable} [command] The command to get the permissions from
+   * @param {Snowflake} [guildID] ID of the guild to get the permissions for,
+   * for global commands and when the guild is not cached
```

Comment:
Looks like I missed two instances of this:
```suggestion
   * ignored when using a {@link GuildApplicationCommandManager}
```